app/frontend/src/components/Actions/Actions.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from "react";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    marginTop: theme.spacing(2),
+    justifyContent: "flex-end",
+    padding: theme.spacing(1),
+    "& :not(:first-child)": {
+      marginLeft: theme.spacing(1),
```

Comment:
```suggestion
      marginInlineStart: theme.spacing(1),
```
`marginInlineStart` and `marginInlineEnd` (but *not* `marginInline`) have good browser compatibility these days, so we have switched to using them in case we ever want to support RtL languages (though there will probably still be plenty of other stuff to fix up!)

Reply:
You could also do `marginBlockStart` instead of `marginTop` if you want, but I don't think it's that important.